3761/1/gem_statisic_fetcher.rb

```diff
@@ -0,0 +1,43 @@
+require './page.rb'
+class GemStatisicFetcher
+  attr_reader :gem_name, :agent
+  attr_reader :stats
+
+  def self.fetch_statistic(gem_name, agent)
```

Comment:
`GemStatis!!!!icFetcher.fetch_statistic` looks like this method name is bad. and you have typo in class name
`GemStatisticFetcher.perform/.call` is much better